magento/inventory
# MSI 2358: disable in store pickup delivery for multishipping

### Description (*)
In-Store Pickup Shipping Method must not be available for Multi-shipping (at least for MVP).
Now it displays like a one more shipping method.
Current pull request will hide it.

### Fixed Issues (if relevant)
1. magento/inventory#2358: Disable In-Store Pickup Delivery for Multishipping

### Manual testing scenarios (*)
1. Put product to the shopping cart
2. Go to shopping cart page
3. Click 'Check Out with Multiple Addresses'
4. Click 'Go to Shipping Information'
5. 'Pick in Store' shipping method is not available in the Shipping Method list

### Contribution checklist (*)
